pkg/plugin/plugin.go

```diff
@@ -98,8 +110,20 @@ func setupVeth(contNetns ns.NetNS, contIfaceName string) (*current.Interface, *c
 		if err != nil {
 			return err
 		}
+
+		containerLink, err := netlink.LinkByName(containerVeth.Name)
+		if err != nil {
+			return fmt.Errorf("failed to lookup %q: %v", containerVeth.Name, err)
+		}
+
+		containerMac := generateRandomMac()
+		err = netlink.LinkSetHardwareAddr(containerLink, containerMac)
```

Comment:
will this prevent duplicates?

Reply:
It won't, but there are not many interfaces in the pod, do you think there is real risk of collisions?

Reply:
But with only 3 octets used, I guess it could happen. Fixing